src/bindgen/ir/function.rs

```diff
@@ -160,12 +200,22 @@ impl Source for Function {
                 }
             }
             cdecl::write_func(out, &func, false, void_prototype);
+
             if !func.extern_decl {
                 if let Some(ref postfix) = postfix {
                     out.write(" ");
                     write!(out, "{}", postfix);
                 }
             }
+
+            if let Some(ref swift_name_macro) = config.function.swift_name_macro {
+                let swift_name = func.swift_name();
+                if !swift_name.is_empty() {
+                    out.write(" ");
```

Comment:
Instead of this, just put the space in the string literal below? These two comments apply to the following block as well.